icons/codesandbox.svg

```diff
@@ -1,3 +1,3 @@
-<svg viewBox="0 0 24 24" rule="img" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 24 24" role="img" xmlns="http://www.w3.org/2000/svg">
```

Comment:
See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon:
> Be minified to a single line with no formatting.